### DISCUSSION TITLE
GIS viewer "color" property

### DISCUSSION THREAD

**Discussion by dmitrysyrchin** [dbeaver/dbeaver#22157] [Q&A] [answered]
DBeaver CE version 21.2.3 and older allowed to display geometry objects from same column with various colors by specifying hex code in column named "color".
![2023-12-12 18_31_11-DBeaver 21 2 3 - Script-240](https://github.com/dbeaver/dbeaver/assets/18523818/f70d0eb0-d30c-471c-9880-afba2628a5ac)
![2023-12-12 18_38_04-DBeaver GIS viewer - Leaflet](https://github.com/dbeaver/dbeaver/assets/18523818/02b7827f-4c33-4c42-8d67-e9cadac77c00)

Since bugfix https://github.com/dbeaver/dbeaver/pull/14377 "colorizing" no longer available and all geometies from same column displayed in default blue. "Color" column is ignored in DBeaver CE version 21.2.4 and later.
![2023-12-12 18_52_35-DBeaver GIS viewer - Leaflet](https://github.com/dbeaver/dbeaver/assets/18523818/95289bc6-c6dd-4d32-8f82-ea64eb27339b)

Is there another way in actual versions of DBeaver CE to "colorizing" geometries from same column in custom colors like it was before bugfix?

**Comment by ShadelessFox** [comment, marked as answer]
Hello @dmitrysyrchin,

It was considered a bug.

Please create a new feature request if you want this feature implemented back: https://github.com/dbeaver/dbeaver/issues
